capsules/src/hts221.rs

```diff
@@ -117,14 +117,16 @@ impl<'a> Hts221<'a> {
                 match self.state.get() {
                     State::Reset => {
                         buffer[0] = REG_AUTO_INCREMENT | CALIB_REG_1ST;
-                        self.i2c.write_read(buffer, 1, 16);
+                        // TODO verify errors
+                        let _ = self.i2c.write_read(buffer, 1, 16);
                         self.state.set(State::Calibrating);
                     }
                     State::Idle(calibration_data, _, _) => {
                         buffer[0] = REG_AUTO_INCREMENT | CTRL_REG1;
                         buffer[1] = 1 << 2 | 1 << 7; // BDU + PD
                         buffer[2] = 1; // ONE SHOT
-                        self.i2c.write(buffer, 3);
+                                       // TODO verify errors
```

Comment:
and here

Reply:
It seems that this is a formatter error. I fixed them and afterwards format fails. I added a newline before some of the comments.